app/routes/joblistings/components/JoblistingEditor.js

```diff
@@ -24,195 +27,249 @@ type Props = {
+    this.props.deleteJoblisting(this.props.joblisting.id).then(() => {
+      this.props.push('/joblistings/');
     });
   };
 
-  if (!isNew && fetching) {
-    return <LoadingIndicator loading />;
+  fetchContacts = (company: SelectInputObject) => {
+    return this.props
+      .fetchCompanyContacts({ companyId: company.value })
+      .then(action => {
+        const responsibleOptions = action.payload.map(contact => ({
+          label: contact.name,
+          value: contact.id
+        }));
+        this.setState({ responsibleOptions });
+      });
+  };
+
+  componentDidMount() {
+    if (!this.props.isNew) {
+      this.fetchContacts(this.props.company);
+    }
   }
 
-  return (
-    <Content>
-      <h1 className={styles.heading}>
-        {!isNew ? 'Rediger jobbannonse' : 'Legg til jobbannonse'}
-      </h1>
-      <Form onSubmit={handleSubmit(onSubmit)}>
-        <Field
-          placeholder="Tittel på jobbannonse"
-          label="Tittel"
-          name="title"
-          component={TextInput.Field}
-          required
-        />
-        <Field
-          placeholder="Bedriften annonsen gjelder"
-          label="Bedrift"
-          name="company"
-          component={SelectInput.AutocompleteField}
-          filter={['companies.company']}
-          onChange={() =>
-            dispatch(
-              change('joblistingEditor', 'responsible', {
-                label: 'Ingen',
-                value: null
-              })
+  render() {
+    const { handleSubmit, isNew, dispatch, fetching = false } = this.props;
+
+    if (!isNew && fetching) {
+      return <LoadingIndicator loading />;
+    }
+
+    return (
+      <Content>
+        <h1 className={styles.heading}>
+          {!isNew ? 'Rediger jobbannonse' : 'Legg til jobbannonse'}
+        </h1>
+        <Form onSubmit={handleSubmit(this.onSubmit)}>
+          <Field
+            placeholder="Title"
+            label="Title"
+            name="title"
+            component={TextInput.Field}
+            required
+          />
+          <Field
+            placeholder="Bedrift"
+            label="Bedrift"
+            name="company"
+            component={SelectInput.AutocompleteField}
+            filter={['companies.company']}
+            onChange={company => {
+              this.fetchContacts(company).then(() => {
+                dispatch(
+                  change('joblistingEditor', 'responsible', {
+                    label: 'Ingen',
+                    value: null
+                  })
+                );
+              });
+            }}
+          />
+          <Field
+            name="jobType"
+            label="Jobbtype"
+            component={SelectInput.Field}
+            placeholder="Jobbtype"
+            simpleValue
+            options={jobTypes}
+          />
+          <Field
+            placeholder="Søknadsfrist"
+            dateFormat="ll"
+            label="Søknadsfrist"
+            showTimePicker={true}
```

Comment:
don't we lint against `={true}` in props? (instead of just writing `showTimePicker`)

Reply:
Looks like we don't :laughing: 

using only `showTimePicker` is much better, agree.